clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1016,12 +1016,16 @@ raw2trace_t::process_syscall_pt(raw2trace_thread_data_t *tdata, uint64_t syscall
             std::to_string(pt2ir_convert_status) + "]";
         return false;
     }
-
+    if (!track_syscall_pt_encodings(tdata, drir.get_ilist())) {
+        return false;
+    }
     /* Convert the DR IR to trace entries. */
+    addr_t sysnum =
+        pt_data->header[dynamorio::drmemtrace::PDB_HEADER_SYSNUM_IDX].sysnum.sysnum;
     std::vector<trace_entry_t> entries;
     trace_entry_t start_entry = { .type = TRACE_TYPE_MARKER,
                                   .size = TRACE_MARKER_TYPE_SYSCALL_TRACE_START,
-                                  .addr = 0 };
+                                  .addr = sysnum };
```

Comment:
The view tool should be updated too, to print the marker value (ditto for _END).

Reply:
Done.